Project-Euler/Problem001.js

```diff
@@ -4,17 +4,24 @@
 Find the sum of all the multiples of 3 or 5 below the provided parameter value number.
 */
 
+//nSum function calculates the sum of first n numbers in the table of number num
+//here n is denoted as frequency
+
+
+//this method uses the nSum function adds the nSum for 3 and 5 but due to this nSum for 15 has been automatically added twice so it needs to be removed once
 const multiplesThreeAndFive = (num) => {
   if (num < 1) throw new Error('No natural numbers exist below 1')
+    num-=1;
+  let sum = 0
 
-  let total = 0
-  // total for calculating the sum
-  for (let i = 1; i < num; i++) {
-    if (i % 3 === 0 || i % 5 === 0) {
-      total += i
+    const nSum = (num,frequency)=>{
```

Comment:
Just `(num, frequency) => frequency*(frequency+1)*num>>1`. Probably should also get a comment to explain what is going on here (summation formula for the arithmetic progression).